applications/GeoMechanicsApplication/custom_elements/three_dimensional_stress_state.cpp

```diff
@@ -0,0 +1,61 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Marjan Fathian
+//                   Richard Faasse
+//
+#include "three_dimensional_stress_state.h"
+#include "custom_utilities/stress_strain_utilities.hpp"
+
+namespace Kratos
+{
+
+Matrix ThreeDimensionalStressState::CalculateBMatrix(const Matrix&         GradNpT,
+                                                     const Vector&         Np,
+                                                     const Geometry<Node>& rGeometry) const
```

Comment:
Does this follow the naming conventions?

Reply:
Both remarks have been addressed.